travis/script-gradle

```diff
@@ -36,7 +36,7 @@ else
       compileTestJava \
       ${documentation:+$SUBMODULE_PREFIX$documentation}
   fi
-  run_gradle "$SUBMODULE_PREFIX"build
+  run_gradle "$SUBMODULE_PREFIX"build publishToMavenLocal
```

Comment:
If we run `publishToMavenLocal` here will that result in all modules getting compiled, even on the cron jobs?  Just want to make sure this won't defeat the purpose of the cron

Reply:
@msridhar asked:
> is there a way to do a local test to see which artifacts would be pushed to a remote repository?

Added in af2329aad5dda5c32e7ac1cddd74e6fed518e3ff. The `publishAllPublicationsToFakeRemoteRepository` task will populate a local repository under `build/maven-fake-remote-repository` that contains exactly what we would upload to a remote repository.